noisepagecontrol/logs/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```

Comment:
add to main gitignore

Reply:
git doesn't allow an empty folder to be committed so the only way to retain the folder is to either
1. add an empty .gitkeep file under this folder
2. add .gitignore like this
imo the second way is more expressive